astro.config.mjs

```diff
@@ -25,7 +26,8 @@ export default defineConfig({
   },
   experimental: { contentLayer: true, serverIslands: true },
   build: {
-    format: "file",
+    format: "directory",

```

Comment:
We are activating the build format to generate a file instead of a directory, as a file is required for Pagefind to return results. this one will break the search